package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "gatsby clean",
     "clean-develop": "gatsby clean && gatsby develop",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop -H 0.0.0.0",
```

Comment:
This will expose the application to the whole network, correct? Is there a way to avoid doing that?

Reply:
ah good point, this is not a problem within a vm but it'd be a problem if someone would to run this directly on the host machine.   I could instead add a new script target?  maybe `develop-expose` or `develop-vm`, something descriptive.  or we can just run gatsby directly.

Reply:
awesome, thanks for the feed back, please see this commit. https://github.com/EnterpriseDB/docs/pull/148/commits/6b495d35695b18ee974717c4d46c2f2ab3999873